spec/models/user_spec.rb

```diff
@@ -98,19 +98,15 @@
       password    = @user.password
       newpassword = "newpassword"
       @user.change_password(password, newpassword)
-      @user.password.should == newpassword
+      expect(@user.password).to eq(newpassword)
     end
 
     it "should raise an error when asked to change user password" do
       password    = "wrongpwd"
       newpassword = "newpassword"
 
-      lambda do
-        @user.change_password(password, newpassword)
-      end.should
-      raise_error(MiqException::MiqEVMLoginError,
-                  "old password does not match current password"
-                 )
+      expect { @user.change_password(password, newpassword) }
+        .to raise_error(MiqException::MiqEVMLoginError)
```

Comment:
Looks much better _and_ see functional change in message of https://github.com/chrisarcand/manageiq/commit/d6546f61a8847a3d6033f2ffef8f8fa423067464